Assets/Scripts/BaseScript.cs

```diff
@@ -52,11 +54,16 @@ void Start()
 
     public void RegisterFailedLanding()
     {
-        rocket.DestroyInstance();
-
-        // Display game over screen
+        // Display game over screen, check to see if current score is a new highscore
+        // if so, updates highscore
         scoreScript.FadeOut();
+        scoreText.GetComponent<Text>().text = scoreScript.GetScore().ToString();
+        scoreScript.CheckIfBest(scoreScript.GetScore());
+        highscoreText.GetComponent<Text>().text = scoreScript.GetHighscore().ToString();
```

Comment:
Minor detail: I wonder if it would might make more sense to have this logic live in score script?